ldi-core/ldi-common/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldi/rdf/formatter/LdiRdfWriter.java

```diff
@@ -17,9 +18,13 @@ public interface LdiRdfWriter {
 
 	static LdiRdfWriter getRdfWriter(LdiRdfWriterProperties properties) {
 		if (Lang.JSONLD.equals(properties.getLang())) {
-			return isBlank(properties.getJsonLdFrame())
-					? new JsonLdPrettyWriter()
-					: new JsonLdFrameWriter(properties);
+			try {
+				return isBlank(properties.getJsonLdFrame())
+						? new JsonLdPrettyWriter()
+						: new JsonLdFrameWriter(properties);
+			} catch (JsonLdError e) {
+				throw new RuntimeException(e);
```

Comment:
My guess is, that this is one of the sonar issues. Perhaps you can create a JsonLdRuntimeError?